frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -17,15 +18,20 @@ import {
   apiVersionForModel,
   k8sCreate,
   Status,
+  GroupVersionKind,
+  ImagePullPolicy,
```

Comment:
nit: keep imports alphabetized. (we're not totally consistent on this, but have been trying to do it to make the imports more manageable)

Reply:
Can we add a linter rule for this (ideally one that supports `--fix`)?

Reply:
I'd love to, but the linter will reorganize all imports. Right now we group between yarn and internal imports. But if we don't care about that, we can enable the linter rule.

Reply:
Ah right. I'd rather not use the linter then.

Reply:
Fixed